src/script/page/LeftSidebar/panels/Conversations/state.ts

```diff
@@ -68,22 +69,51 @@ export enum SidebarTabs {
   PREFERENCES,
 }
 
+export const SidebarOpenStatus = {
+  OPEN: 'OPEN',
+  CLOSED: 'CLOSED',
+  MANUAL_OPEN: 'MANUAL_OPEN',
+  MANUAL_CLOSED: 'MANUAL_CLOSED',
+} as const;
```

Comment:
As @svitovyda suggested, this can be simplified to 

```
OPEN, // When the user has decided to open manually
CLOSED, // when the user has decided to manually close
AUTO, // Will automatically adapt to the screen size
```